src/Lykke.Service.KucoinAdapter.Services/KucoinInstrument.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace Lykke.Service.KucoinAdapter.Services
+{
+    public sealed class KucoinInstrument
+    {
+        private readonly IReadOnlyDictionary<string, string> _rename;
+        private readonly IReadOnlyDictionary<string, string> _inverted;
+
+        private static readonly string[] DefaultCurrencies = {
```

Comment:
I't sbetter to move this array to settings, so new currencoes can be added just by restarting the app.